app/assets/v2/css/base.css

```diff
@@ -1674,3 +1674,70 @@ div.busyOverlay {
 .g-text-left {
   text-align: left;
 }
+
+/* Bubble */
```

Comment:
Should we move this to `dashboard.css` ?

Reply:
I think we can reuse this bubble for other places